devicon.json

```
@@ -7756,6 +7756,26 @@
         "color": "#F05138",
         "aliases": []
     },
+    {
+       "name": "swagger",
+       "altnames": ["Swagger"],
+       "tags": ["development"],
+       "versions": {
+       "svg": ["original", "plain"],
+       "font": ["original", "plain"]
+       },
+       "color": "#173647",
+       "aliases": [
+           {
+           "base": "original",
+           "alias": "original-wordmark"
+           },
+           {
+           "base": "plain",
+           "alias": "plain-wordmark"
+           }
+       ]
+    },
```

I'd suggest some changes.

```suggestion
    {
       "name": "swagger",
       "altnames": [],
       "tags": [
           "development",
           "api",
           "documentation",
           "tool"
       ],
       "versions": {
           "svg": [
               "original",
               "plain"
           ],
           "font": [
               "original",
               "plain"
           ]
       },
       "color": "#173647",
       "aliases": [
           {
               "base": "original",
               "alias": "original-wordmark"
           },
           {
               "base": "plain",
               "alias": "plain-wordmark"
           }
       ]
    },
```